tests/tests_utils.py

```diff
@@ -71,3 +81,48 @@ def test_local_file_sample_size(self):
 # TODO: test normalize_mime_type
 # TODO: test plugin_name_by_mime_type
 # TODO: test plugin_name_by_uri
+
+
+class UtilsDecompressTestCase(unittest.TestCase):
+
+    def setUp(self):
+        self.contents = six.b('Ahoy')
+        self.temp = tempfile.TemporaryDirectory()
```

Comment:
All the tests are failing here. I've replaced `self.tmp` with `self.temp` (was receiving a `NameError`) but they still fail:

```
======================================================================
ERROR: test_decompress_with_bz2 (tests.tests_utils.UtilsDecompressTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/turicas/projects/rows/tests/tests_utils.py", line 99, in test_decompress_with_bz2
    decompressed = rows.utils.decompress(compressed)
  File "/home/turicas/projects/rows/rows/utils.py", line 359, in decompress
    raise RuntimeError(msg)
RuntimeError: Couldn't identify file mimetype, or lzma module isn't available

======================================================================
ERROR: test_decompress_with_gz (tests.tests_utils.UtilsDecompressTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/turicas/projects/rows/tests/tests_utils.py", line 107, in test_decompress_with_gz
    self.assertEqual(self.contents, decompressed.read())
  File "/home/turicas/software/pyenv/versions/3.6.2/lib/python3.6/gzip.py", line 272, in read
    self._check_not_closed()
  File "/home/turicas/software/pyenv/versions/3.6.2/lib/python3.6/_compression.py", line 14, in _check_not_closed
    raise ValueError("I/O operation on closed file")
ValueError: I/O operation on closed file

======================================================================
ERROR: test_decompress_with_incompatible_file (tests.tests_utils.UtilsDecompressTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/turicas/projects/rows/tests/tests_utils.py", line 126, in test_decompress_with_incompatible_file
    with self.assertRaises():
TypeError: assertRaises() missing 1 required positional argument: 'expected_exception'

======================================================================
ERROR: test_decompress_with_lzma (tests.tests_utils.UtilsDecompressTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/turicas/projects/rows/tests/tests_utils.py", line 112, in test_decompress_with_lzma
    with lzma.open(compressed) as compressed_handler:
  File "/home/turicas/software/pyenv/versions/3.6.2/lib/python3.6/lzma.py", line 302, in open
    preset=preset, filters=filters)
  File "/home/turicas/software/pyenv/versions/3.6.2/lib/python3.6/lzma.py", line 120, in __init__
    self._fp = builtins.open(filename, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpr5s7bse8/test.lzma'

======================================================================
ERROR: test_decompress_with_xz (tests.tests_utils.UtilsDecompressTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/turicas/projects/rows/tests/tests_utils.py", line 120, in test_decompress_with_xz
    with lzma.open(compressed) as compressed_handler:
  File "/home/turicas/software/pyenv/versions/3.6.2/lib/python3.6/lzma.py", line 302, in open
    preset=preset, filters=filters)
  File "/home/turicas/software/pyenv/versions/3.6.2/lib/python3.6/lzma.py", line 120, in __init__
    self._fp = builtins.open(filename, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpz5l7xm1q/test.gz'

----------------------------------------------------------------------
Ran 184 tests in 0.848s
```

Are the tests passing in your machine?